VideoLocker/src/main/java/org/edx/mobile/module/prefs/PrefManager.java

```diff
@@ -392,6 +400,7 @@ public void setLastCourseStructureFetch(String courseId, long timestamp) {
         public static final String AppSettingNeedSyncWithParse = "AppSettingNeedSyncWithParse";
         public static final String UserPrefVideoModel = "UserPrefVideoModel";
         public static final String LAST_COURSE_STRUCTURE_FETCH = "LastCourseStructureFetch";
+        public static final String VIDEOS_CACHE_RESTORED = "VideosCacheRestored";
```

Comment:
Should this be documented here. I see that the keys (and the accessor and setter methods) aren't documented here, but this may be a temporary key so it may be worth explicitly documenting it's purpose and scope here to aid in it's potential future removal.